assets/airports/egnm.json

```diff
@@ -10,251 +10,2015 @@
     "ctr_radius": 32,
     "ctr_ceiling": 8500,
     "initial_alt": 3000,
```

Comment:
I think 7000 is a more accurate initial altitude

Reply:
forgot to change that 👍